sw/device/lib/testing/json/provisioning_data.h

```diff
@@ -91,17 +93,16 @@ UJSON_SERDE_STRUCT(ManufRmaTokenPersoDataOut, \
 // clang-format on
 
 /**
- * Data exported during device (attestation) certificate personalization.
- *
- * TODO(#19455): replace the certificate fields with actual X.509 certificates
- * instead of just the public keys.
+ * Certificates exported during device personalization.
  */
 // clang-format off
 #define STRUCT_MANUF_CERT_PERSO_DATA_OUT(field, string) \
     field(uds_certificate, uint8_t, 596) \
     field(uds_certificate_size, size_t) \
-    field(cdi_0_certificate, ecc_p256_public_key_t) \
-    field(cdi_1_certificate, ecc_p256_public_key_t)
+    field(cdi_0_certificate, uint8_t, 634) \
```

Comment:
Should the size use the the constant defined in the generated header instead of being hardcoded? Sane for UDS and CDI_1

Reply:
Unfortunately, ujson infra does not allow the use of constants like that, but I added some static asserts to ensure things stay in sync.